pwem/protocols/protocol_export/protocol_export_DB.py

```diff
@@ -0,0 +1,233 @@
+# **************************************************************************
+# *
+# * Authors:     Amaya Jimenez (ajimenez@cnb.csic.es)
+# *              Marta Martinez (mmmtnez@cnb.csic.es)
+# *              Roberto Marabini (roberto@cnb.csic.es)
+# *
+# * Unidad de  Bioinformatica of Centro Nacional de Biotecnologia , CSIC
+# *
+# * This program is free software; you can redistribute it and/or modify
+# * it under the terms of the GNU General Public License as published by
+# * the Free Software Foundation; either version 2 of the License, or
+# * (at your option) any later version.
+# *
+# * This program is distributed in the hope that it will be useful,
+# * but WITHOUT ANY WARRANTY; without even the implied warranty of
+# * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# * GNU General Public License for more details.
+# *
+# * You should have received a copy of the GNU General Public License
+# * along with this program; if not, write to the Free Software
+# * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
+# * 02111-1307  USA
+# *
+# *  All comments concerning this program package may be sent to the
+# *  e-mail address 'scipion@cnb.csic.es'
+# *
+# **************************************************************************
+
+
+import os
+
+import pyworkflow.protocol.params as params
+from pyworkflow import VERSION_1_2
+
+from pwem.convert.atom_struct import fromPDBToCIF, fromCIFTommCIF
+from pwem.emlib.image import ImageHandler
+from pwem.protocols import EMProtocol
+from pwem.objects import FSC
+from pyworkflow.utils.path import copyFile
+
+class ProtExportDataBases(EMProtocol):
```

Comment:
This class name (and protocol?) seems duplicated

Reply:
The protocol pwem/protocols/protocol_export/protocol_export_emdb.py doesn't exist anymore, it was replaced by pwem/protocols/protocol_export/protocol_export_DB.py.